### ISSUE TITLE
ApplicationInspector does not render HTML report page properly if run outside its folder

### ISSUE THREAD

**Issue by localacct** [microsoft/ApplicationInspector#93]
**Describe the bug**
When running AppInspector.dll from outside its folder to generate HTML report, the page does not get rendered properly due to relative paths that reference where AppInspector.dll is located.

````
<link rel="stylesheet" href="html/resources/css/bootstrap.min.css"
````

The page will try to look for these files relative to where the command line was run (example the folder where I store the source code files, which is **/path/to/source_code/files**).

````
dotnet /path/to/AppInspector.dll analyze -s /path/to/source_code/files/file.c
````

**To Reproduce**
Run the command outside of AppInspector.dll's folder.

````
dotnet /path/to/AppInspector.dll analyze -s /path/to/source_code/files/file.c
````

**Expected behavior**
The HTML report page should be rendered properly

**Screenshots**
<img width="634" alt="rendered incorrectly" src="https://user-images.githubusercontent.com/39975764/72859704-b806c580-3cff-11ea-9794-b4b13b2f3f6f.png">

**Desktop (please complete the following information):**
 - OS: Mac OS 10.14.6 Mojave
 - Browser Google Chrome
 - Version 79.0.3945.130

**Smartphone (please complete the following information):**

**Additional context**

**Comment by guyacosta** [comment]
Will look into addressing.  In the meantime, just run it from the app folder and provide a path to your source to scan.

**Comment by guyacosta** [comment]
I have fix for it that's working and will get into the code base later today.

**Comment by guyacosta** [comment]
Fixed in latest code base #94.  Release will follow.
